index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/nav-bar.css">
+  <link rel="stylesheet" href="./styles/styles.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+  <title>Global Training Courses - Home</title>
+</head>
+<body>
+  <header>
+    <nav class="nav-mobile">
+      <img class="menu-button" id="menu-button" src="./assets/hamburger-menu.svg" alt="menu">
+      <div class="menu" id="menu">
+        <div>
+          <img id="close-button" src="./assets/close.png" alt="close">
+        </div>
+        <ul>
+          <li><a href="about.html">About Us</a></li>
+          <li><a href="#topics">Topics</a></li>
+          <li><a href="#courses">Courses</a></li>
+          <li><a href="#">Log In</a></li>
+          <li><a href="#">Sign Up</a></li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
- [ ] You are missing the logo, kindly follow the project design by adding a logo as the picture below
    
![no-logo1](https://user-images.githubusercontent.com/66097371/164949577-dc2c5e1a-7c0f-463b-8552-6566235792d6.png)